### DISCUSSION TITLE
Catalog of Life issue

### DISCUSSION THREAD

**Discussion by mattfossen** [Symbiota/community-board#464] [Problems/Bugs]
Hi all,

I've been having an issue with the taxonomy cleaner and Catalog of Life: no matter what scientific names are submitted, CoL will NEVER find a match. This seems to be a fairly recent problem (going back maybe 2-3 months), as it used to work fairly well. Any thoughts?

Thanks!

**Comment by themerekat** [comment]
Hi @mattfossen , what portal are you working in?

**Reply by mattfossen** [reply]
It's one I'm self-hosting to manage my personal insect collection; it's not affiliated with any real portals so i know support is limited at best.

**Reply by themerekat** [reply]
@mattfossen , ah ok! The Catalog of Life API was taken down in deference to the ChecklistBank API several months ago, causing this breakage. We have since made necessary changes to the Symbiota codebase to leverage the ChecklistBank API instead of CoL, so if you update your code, you should fix the problem.
That being said, there might be some schema changes since the last update. Make sure you run the schema updates if you update your code!

**Reply by mattfossen** [reply]
Thanks so much! Updating the taxonomy harvester and the database schema did the trick!

**Reply by themerekat** [reply]
Woot! Glad to hear

**Reply by nfranz** [reply]
@mattfossen have you considered joining https://ecdysis.org/collections/index.php

**Reply by mattfossen** [reply]
@nfranz I thought about it when I was initially trying to get everything set up, but to be honest a lot of my data is quite poor (old specimens I collected as a teen with no locality data, IDs that are largely guesswork, etc.) and the specimen photos vary wildly in quality as I was just learning how to take them.

**Reply by nfranz** [reply]
Got it, thanks @mattfossen. If there's ever a subset that you'd like to have on a public portal, please reach out.
